pkg/sql/executor.go

```diff
@@ -1447,18 +1463,20 @@ func (e *Executor) execStmtInOpenTxn(
 		}
 
 		sessionEventf(session, "ERROR: %v", err)
-		return Result{}, err
+		return err
 	}
 
-	tResult := &traceResult{tag: result.PGTag, count: -1}
-	switch result.Type {
-	case parser.RowsAffected:
-		tResult.count = result.RowsAffected
-	case parser.Rows:
-		tResult.count = result.Rows.Len()
-	}
-	sessionEventf(session, "%s done", tResult)
-	return result, nil
+	// TODO(tohlson): support tracing
```

Comment:
I assume that it's fine if this tracing just moves to the result writer implementation?